.gitlab-ci.yml

```diff
@@ -132,6 +132,53 @@ test:release:
     reports:
       junit: "**/build/test-results/testReleaseUnitTest/*.xml"
 
+# We need separated steps for tracing module tests as these cannot be run with the CI visibility agent enabled. Also they need to be executed with
+# the daemon capability as the worker threads are running as a daemon.
+
+test:trace-debug:
+  tags: [ "arch:amd64" ]
+  image: $CI_IMAGE_DOCKER
+  stage: test
+  timeout: 1h
+  variables:
+    DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED: "false"
```

Comment:
btw is this flag still needed given that we clean up env variable in the `DDSpecification` and default value of this one in `Config` is `false`?

Reply:
yes I am not sure where exactly before that is still used...before the Config is even created. as you could see in my tests I assert that there is no more any DD_ property in any System or System.env and still this poses problems. Alessio from APM team recommended me to add it like this to make sure.

Reply:
We can remove it and see if it is still needed. Maybe it is not anymore.